site/layouts/shortcodes/note.html

```diff
@@ -1,6 +1,3 @@
 <aside class="note">
-    <div class="note-icon">
-        {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
```

Comment:
This partial is just an empty svg file. I removed it because it would render as <p></p>`.